### PR TITLE
remove/revert some rules + revert baptiste regen burst healing amount

### PR DESCRIPTION
**Pull request by ieToastie**
Edits a few constants, finishes the rest of the tweaks regarding next patch's balance updates, comments excess rules to reduce overall script count and workshop load.

### CODE REVIEW

File: src/lobby/dev_lobby.opy [hunk 1]
```diff
@@ -51,8 +51,7 @@ settings {
                 "secondaryFireMaximumTime%": 66
             },
             "doomfist": {
-                "ability1Cooldown%": percent(ADJ_DOOMFIST_SLAM_CD/OW2_DOOMFIST_SLAM_CD),
-                "ability2Cooldown%": percent(ADJ_DOOMFIST_POWER_BLOCK_COOLDOWN/OW2_DOOMFIST_POWER_BLOCK_COOLDOWN),
+                "ability1Cooldown%": percent(ADJ_DOOMFIST_POWER_BLOCK_COOLDOWN/OW2_DOOMFIST_POWER_BLOCK_COOLDOWN),
```
Comment (MaxwellJung) [inline src/lobby/dev_lobby.opy]:
```suggestion
                "ability1Cooldown%": percent(ADJ_DOOMFIST_SLAM_CD/OW2_DOOMFIST_SLAM_CD),
                "ability2Cooldown%": percent(ADJ_DOOMFIST_POWER_BLOCK_COOLDOWN/OW2_DOOMFIST_POWER_BLOCK_COOLDOWN),
```
I thought ability 1 = slam and ability 2 = block in ow2

Comment (MaxwellJung) [reply src/lobby/dev_lobby.opy]: Also I would comment out the line instead of deleting it so eaiser to revert later.

File: src/lobby/lobby.opy [hunk 2]
```diff
@@ -79,7 +79,6 @@ settings {
                 "secondaryFireMaximumTime%": 66
             },
             "doomfist": {
-                "ability1Cooldown%": percent(ADJ_DOOMFIST_SLAM_CD/OW2_DOOMFIST_SLAM_CD),
```
Comment (MaxwellJung) [inline src/lobby/lobby.opy:82]: see above comment

**Review by MaxwellJung — CHANGES_REQUESTED** [review]
Check doomfist cooldown logic in dev_lobby.opy and lobby.opy

Also, let's try to replace any magic numbers with constants in the codebase as we encounter them.

**Review by MaxwellJung — APPROVED** [review]
ADJ_BAPTISTE_LAMP_HP isn't used in the codebase, so changing it has no impact yet.

**Review by ieToastie — COMMENTED** [review]
Looks good to me, will add constants where applicable from here on out

**Pull request merged**